routes/api.js

```diff
@@ -58,6 +58,13 @@ router.get('/local/action/pause', map2dorita('local', 'pause'));
 router.get('/local/action/dock', map2dorita('local', 'dock'));
 router.get('/local/action/resume', map2dorita('local', 'resume'));
 
+
+router.post('/local/action/start', map2dorita('local', 'start'));
+router.post('/local/action/stop', map2dorita('local', 'stop'));
+router.post('/local/action/pause', map2dorita('local', 'pause'));
+router.post('/local/action/dock', map2dorita('local', 'dock'));
+router.post('/local/action/resume', map2dorita('local', 'resume'));
+
```

Comment:
instead of add this four. I would like to have only one new post entry like:
`router.post('/local/action/cleanRoom', map2dorita('local', 'start', true));`
note that the action remais start and the `true` last param. This value as true allows us to remove yours other changes below